src/main/java/mate/academy/springboot/datajpa/service/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.springboot.datajpa.service.mapper;
+
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class CategoryMapper {
```

Comment:
Maybe it's a good idea to create generic interface that contains 2 method `toDto` and `toModel` and implement it in your mappers? Don't forget about SOLID principles